packages/terra-divider/tests/wdio/divider-spec.js

```diff
@@ -0,0 +1,16 @@
+/* global browser, Terra */
+const viewports = Terra.viewports('tiny', 'medium');
```

Comment:
Let's only take a screenshot at the `medium` size. This component doesn't change at different screen widths.